meta.go

```diff
@@ -438,7 +442,57 @@ func main() {
 			},
 			Flags: []cli.Flag{jsonValueFlag},
 		},
+		{
+			Name:  "lastSuccessfulMeta",
+			Usage: "Fetch lastSuccessfulMeta from an external job",
+			ArgsUsage: `job-description
+      job-description may be sd@pipelineid:jobname or just jobname for same pipeline
+`,
+			Action: func(c *cli.Context) error {
+				if len(c.Args()) < 1 {
+					return cli.ShowCommandHelp(c, c.Command.Name)
+				}
+				jobDescription, err := fetch.ParseJobDescription(
+					lastSuccessfulMetaRequest.DefaultSdPipelineId, c.Args().First())
+				if err != nil {
+					log.Fatal(err)
+				}
+				meta, err := lastSuccessfulMetaRequest.FetchLastSuccessfulMeta(http.DefaultTransport, jobDescription)
+				if err != nil {
+					log.Fatal(err)
+				}
+				metaFile = jobDescription.External
+				metaFilePath := filepath.Join(metaSpace, metaFile+".json")
+				err = ioutil.WriteFile(metaFilePath, meta, 0666)
+				if err != nil {
+					log.Fatal(err)
+				}
+				return nil
+			},
+			Flags: []cli.Flag{
+				cli.StringFlag{
+					Name:        "sd-token, t",
+					Usage:       "Set the SD_TOKEN to use in SD API calls",
+					EnvVar:      "SD_TOKEN",
+					Destination: &lastSuccessfulMetaRequest.SdToken,
+				},
+				cli.StringFlag{
+					Name:        "sd-api-url, u",
+					Usage:       "Set the SD_API_URL to use in SD API calls",
+					EnvVar:      "SD_API_URL",
+					Destination: &lastSuccessfulMetaRequest.SdApiUrl,
+				},
+				cli.Int64Flag{
+					Name:        "sd-pipeline-id, p",
+					Usage:       "Set the SD_PIPELINE_ID for job description",
```

Comment:
set the SD_PIPELINE_ID of the job for fetching last successful meta. 
`job description` is a little vague and specific to the implementation details

Reply:
Done